### DISCUSSION TITLE
Search Censorship

### DISCUSSION THREAD

**Discussion by Morasithil** [searxng/searxng#2744] [Q&A]
When i make a searchrequest with the following input "WTC7 explosion" searxng responds with a notification as follows:

"Sorry! we didn't find any results. Please use another query or search in more categories."

I've tried this with four searxng instances and only one of them showed any results at all...
The instance https://searx.fmac.xyz/ (hosted in switerland) and https://searx.work/ (hosted in luxembourg) aswell as https://searx.prvcy.eu/ (hosted in finland) do not show any search results for  "WTC7 explosion".
The instance https://searx.tuxcloud.net/ (hosted in czechia) does show search results for "WTC7 explosion".

What's going on here? This clearly makes an impression of censorship, if there are other reasons i'm listening.
It's really sad how most search engines push propaganda like wikipedia saying the WTC7 explosion is a conspiracy theory when it's obviously a controlled demolition and not some random fire perfectly collapsing the building.
It's sad that over 20 years after 9/11 the internet still pushed lies on top of searchresults, atleast with the searchengines i've tried.

**Comment by mrpaulblack** [comment]
Dupe of https://github.com/searxng/searxng/issues/2739 ; Closing.
